provider/datagen/src/transform/cldr/cldr_serde/numbers.rs

```diff
@@ -86,12 +86,24 @@ impl<'de> Visitor<'de> for DecimalFormatVisitor {
     }
 }
 
+#[derive(PartialEq, Debug, Deserialize)]
+pub struct CurrencyFormats {
```

Comment:
would `CurrencyFormatsPattern` be a more accurate name?

Reply:
Also the comments should probably say "A standard pattern" and "A standard alphaNextToNumber pattern", as there are no maps here

Reply:
good catch, done